diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index 036b044..2a32197 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -10,7 +10,7 @@
 'use strict';
 
 const PDFJS = require('./lib/pdf.js');
-const {PNG} = require('pngjs');
+const PNG = () => {};
 
 function render_page(pageData) {
     //check documents https://mozilla.github.io/pdf.js/typedoc/interfaces/PageViewport.html
